extract_unique_sensors_01.py

```diff
@@ -0,0 +1,28 @@
+from urllib.request import urlopen
+
+sensors_data = urlopen('http://monctl.devm.camlab.kat.ac.za/kat/doc/manuals/sensors/SensorList.txt')
+lines = sensors_data.readlines()
+
+
+def sensors(lines):
```

Comment:
Function name needs to be declarative. 
You should write code in such a way that you would be able to explain it to a 5-year-old.